gatsby_fp/src/components/fieldComponents/intellCont.js

```diff
@@ -0,0 +1,195 @@
+                    dty_pub: ${element.dty_pub}<br> `:
+                    ``
+                  }
+                  ${element.dtm_acc ? `
+                    dtm_acc: ${element.dtm_acc}<br> `:
+                    ``
+                  }
+                  ${element.dtd_acc ? `
+                    dtd_acc: ${element.dtd_acc}<br> `:
+                    ``
+                  }
+                  ${element.dty_acc ? `
+                    dty_acc: ${element.dty_acc}<br> `:
+                    ``
+                  }
+                  ${element.dtm_expsub ? `
+                    dtm_expsub: ${element.dtm_expsub}<br> `:
+                    ``
+                  }
+                  ${element.dtd_expsub ? `
+                    dtd_expsub: ${element.dtd_expsub}<br> `:
+                    ``
+                  }
+                  ${element.dty_expsub ? `
+                    dty_expsub: ${element.dty_expsub}<br> `:
+                    ``
+                  }
+                  ${element.dtm_sub ? `
+                    dtm_sub: ${element.dtm_sub}<br> `:
+                    ``
+                  }
+                  ${element.dtd_sub ? `
+                    dtd_sub: ${element.dtd_sub}<br> `:
+                    ``
+                  }
+                  ${element.dty_sub ? `
+                    dty_sub: ${element.dty_sub}<br> `:
+                    ``
+                  }
+                  ${element.doi ? `
+                    doi: ${element.doi}<br> `:
+                    ``
+                  }
+                  ${element.abstract ? `
+                    abstract: ${element.abstract}<br> `:
+                    ``
+                  }
+                  ${element.full_text ? `
+                    full_text: ${element.full_text}<br> `:
+                    ``
+                  }
+                  ${element.web_address ? `
+                    web_address: ${element.web_address}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.faculty_name ? `
+                    faculty_name: ${element.intellcont_auth.faculty_name}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.fname ? `
+                    intellcont_auth.fname: ${element.intellcont_auth.fname}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.lname ? `
+                    intellcont_auth.lname: ${element.intellcont_auth.lname}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.role ? `
+                    intellcont_auth.role: ${element.intellcont_auth.role}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.institution ? `
+                    intellcont_auth.institution: ${element.intellcont_auth.institution}<br> `:
+                    ``
+                  }
+                  ${element.intellcont_auth.student_level ? `
```

Comment:
These all hold a similar shape, with similar rendering logic. Perhaps we can loop through the properties of the object, and render those that are set?